### PR TITLE
Feature/remove django choices

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

**Review by SilviaAmAm — APPROVED** [review]

File: tests/test_clients.py [hunk 1]
```diff
@@ -1531,7 +1531,7 @@ def test_full_update_gebruiksrechten(self):
 
         # Create gebruiksrechten
         gebruiksrechten_data = {
-            "informatieobject": f"https://testserver/api/v1/documenten/{document.uuid}",
```
Comment (joeribekker) [inline tests/test_clients.py:1534]: Why was this changed? I prefer not to have specific URLs.

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: tests/test_clients.py (hunk 1 above)
Comment (annashamray) [reply tests/test_clients.py:1534]: These "https://testserver/..." urls fail when CI runs with turned on URL mapping, because this pattern is not known for set upt UrlMapping data. 
TBH I don't know why it worked before, but other Gebruiksrechten tests have "https://drc.utrechtproeftuin.nl/..." urls so I just adjusted these two tests to match

Comment (joeribekker) [reply tests/test_clients.py:1534]: Ok, then this will be moved to a new issue.

**Review by joeribekker — APPROVED** [review]
